DSN/backend/src/main/java/org/gfa/dsn/utils/SecurityConfig.java

```diff
@@ -53,6 +53,7 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity) throws Excepti
             "/v3/api-docs/**",
             "/v3/api-docs.yaml",
             "/api/v1/auth/**",
+
```

Comment:
Please remove this unnecessary empty line.

Reply:
Done